### PR TITLE
ci: restart portmap service on CI nodes

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-1.16-netnext

**Comment by nebril** [comment]
test-1.16-netnext

**Comment by nebril** [comment]
test-1.19-5.4

**Comment by nebril** [comment]
test-gke

**Comment by nebril** [comment]
test-1.20-4.19

**Comment by nebril** [comment]
test-1.21-4.9

**Comment by nebril** [comment]
test-runtime

**Comment by nebril** [comment]
test-me-please

File: test/vagrant-ci-start.sh [hunk 1]
```diff
@@ -4,6 +4,10 @@ set -e
 
 K8S_NODES=${K8S_NODES:-2}
 
+echo "restarting portmap and nfs-kernel-server services to combat nfs server issues"
+systemctl restart portmap.service || true
+systemctl restart nfs-kernel-server.service || true
```
Comment (pchaigno) [inline test/vagrant-ci-start.sh:9]: Why `|| true`? Don't we want to fail if restarting the service fails?

Comment (nebril) [reply test/vagrant-ci-start.sh:9]: Restarting `nfs-kernel-server` can actually fail while nfs server is running with this error message:
```
Job for nfs-server.service canceled.
```
, which means that everything is fine and we should continue.

**Review by pchaigno — APPROVED** [review]

**Review by nbusseneau — APPROVED** [review]

**Comment by nebril** [comment]
All builds affected by this change managed to go through provisioning step. Merging this, we will monitor new builds for this failure.

**Pull request merged**